components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/pages/cdmf.page.device-organization/public/js/client-script.js

```diff
@@ -0,0 +1,64 @@
+var nodes = null;
+var edges = null;
+var backendApiUrlNodes = "/api/device-mgt/v1.0/device-organization/1.0.0/nodes";
+var successCallbackNodes = function (dataNodes) {
+    if (dataNodes) {
+        console.log(JSON.parse(dataNodes));
+        nodes = JSON.parse(dataNodes);
+        console.log(nodes);
+    }
+};
+invokerUtil.get(backendApiUrlNodes, successCallbackNodes, function (messageNodes) {
+});
+
+var backendApiUrlEdges = "/api/device-mgt/v1.0/device-organization/1.0.0/edges";
+var successCallbackEdges = function (dataEdges) {
+    if (dataEdges) {
+        console.log(JSON.parse(dataEdges));
```

Comment:
these console.logs are not needed IMO. 
Best practise is not to log issues or status as console logs.

Reply:
Noted @Kamidu 
I believe those have remained over from testing work. Will remove those.